CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ straightforward as possible.
 
 ### Changed
 - dependencies update
+- rewrite feed on streams
```

Comment:
This is for customer, not for devs

Should be more like `updating feed rendering w/ new mechanics`

Reply:
Right, thanks